lib/success.ts

```diff
@@ -62,6 +63,39 @@ async function findOrCreateVersion(config: PluginConfig, context: GenerateNotesC
   return newVersion;
 }
 
+async function editIssueFixVersions(config: PluginConfig, context: GenerateNotesContext, jira: JiraClient, newVersionName: string, releaseVersionId: string, issueKey: string): Promise<void> {
+  try {
+    context.logger.info(`Adding issue ${issueKey} to '${newVersionName}'`);
+    if (!config.dryRun) {
+      await jira.issue.editIssue({
+        issueKey,
+        issue: {
+          update: {
+            fixVersions: [{
+              add: { id: releaseVersionId },
+            }],
+          },
+        },
+      });
+    }
+  } catch (err) {
+    const allowedStatusCodes = [400, 404];
```

Comment:
It could be good to move that in config or wherever else than redeclare it for each promise :)
Your PR could be a good occasion to do it :)